src/components/AccountDrawer/MiniPortfolio/Activity/parseLocal.ts

```diff
@@ -158,6 +158,7 @@ export function transactionToActivity(
       timestamp: (details.confirmedTime ?? details.addedTime) / 1000,
       from: details.from,
       nonce: details.nonce,
+      cancelled: details.cancelled,
```

Comment:
I can see that this is working, but I'm confused. It looks like this is a local transaction property, but the types (and my knowledge of our architecture) make me think this should be a remote (gql) property only.

Reply:
The backend does not tell us whether a tx was cancelled or not. It only provides us with whatever tx made it on chain. So `cancelled` is a local transaction property that we set when we find out that a pending tx was replaced by another tx. Lmk if I'm misunderstanding your question